apps/explorer/src/app/components/jump-to-block/index.tsx

```diff
@@ -20,8 +20,20 @@ export const JumpToBlock = () => {
 
   return (
     <form onSubmit={handleSubmit}>
-      <input type={'tel'} name={'blockNumber'} placeholder={'Block number'} />
-      <input type={'submit'} value={'Go'} />
+      <label
+        htmlFor="block-input"
+        className="block uppercase text-h5 font-bold"
+      >
+        Jump to block
+      </label>
+      <input
+        id="block-input"
+        type={'tel'}
```

Comment:
No need for curly braces here. Same goes for other props in this PR

```suggestion
        type='tel'
```